web/src/layouts/CustomerLayout.jsx

```diff
@@ -1,9 +1,36 @@
-import { Outlet } from "react-router-dom";
+import { Layout } from 'antd';
+import React from 'react';
+import Sidebar from '../components/Sidebar';
+import colors from '../constants/Colors';
+// import { Outlet } from "react-router-dom";
 
-export default function CustomerLayout() {
+const { Content } = Layout;
+
+const CustomerLayout = () => {
     return (
-        <>
-            <Outlet />
-        </>
-    )
-}
\ No newline at end of file
+        <Layout style={{
+            background: `linear-gradient(45deg, ${colors.gradientStart}, ${colors.gradientEnd})`,
+            padding: '50px',
+            margin: '0px',
+            width: '100%',
+            height: '100vh',
+        }}>
+            <Layout style={{ boxShadow: `0 4px 8px ${colors.shadow}`, borderRadius: 30 }}>
+                <Sidebar />
+
+                <Layout style={{
+                    borderTopRightRadius: 30,
+                    borderBottomRightRadius: 30,
+                    background: colors.background,
+                    padding: '20px',
+                }}>
+                    <Content>
+                        {/* <Outlet />  */}
```

Comment:
Mở comment component này ra để có thể hiển thị các component con nha @truongnm-0478 
```suggestion
                        <Outlet />
```